object-manipulation-1/src/get-values.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-unused-vars */
+
+function getValues(object) {
+  const values = [];
+  if (object === {}) return values;

```

Comment:
objects are only ever `===` to themselves, so this `if` never happens. you can remove it

Reply:
Okay makes sense. Is it because they're a ref data type?